datayoga/blocks/remove_field/block.py

```diff
@@ -14,5 +14,6 @@ def init(self):
     def run(self, data: Any, context: Context = None) -> Any:
         logger.debug(f"Running {self.get_block_name()}")
 
-        del data[self.properties["field"]]
+        for row in data:
+            del row[self.properties["field"]]
```

Comment:
should we manipulate the data itself or cloning it first?

Reply:
good question. maybe this should happen outside the block so it is generic? only downside is the performance penalty

Reply:
yes, probably outside of the block is the right approach.

Reply:
let's open a separate ticket to do it there

Reply:
https://github.com/datayoga-io/datayoga-py/issues/33